File: src/SamplePublishing/RA.SamplesForDocumentation/SampleModels/Organization.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RA.SamplesForDocumentation.SampleModels
{
	public class Organization : BaseModel
	{
		public string OrganizationType { get; set; } = "CredentialOrganization";

		public string ImageUrl { get; set; }
		/// <summary>
		/// The type of the described agent.
		/// Must provide valid organization types.
		/// May provide with or without the orgType namespace
		/// orgType:CertificationBody
		/// </summary>
		public List<string> AgentType { get; set; }

		public List<string> Keywords { get; set; }
	}
}

File: src/SamplePublishing/VisualBasicPublishingSamples/SampleModels/Organization.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

'Namespace VisualBasicSamplePublishing
Public Class Organization
	Inherits BaseModel

	Public Property OrganizationType As String = "CredentialOrganization"
	Public Property ImageUrl As String
	Public Property Email As List(Of String)
	Public Property AgentType As List(Of String)
	Public Property Keywords As List(Of String)
	Public Property AgentSectorType As String = "PrivateNonProfit"

End Class
'End Namespace
